contracts/examples/nft/scripts/localnet.sh

```
@@ -0,0 +1,8 @@
+<<<<<<< HEAD
```

Shellcheck (0.10.0)

[error] Couldn't parse this here string. Fix to allow more checks.

(SC1073)